packages/contracts-bedrock/test/DeployOPChain.t.sol

```diff
@@ -3,6 +3,15 @@ pragma solidity 0.8.15;
 
 import { Test } from "forge-std/Test.sol";
 
+import { DeploySuperchainInput, DeploySuperchain, DeploySuperchainOutput } from "scripts/DeploySuperchain.s.sol";
```

Comment:
Similar to how `DeploySuperchain.s.sol` is thoroughly commented because we use that script to document the architecture that we're using, the same goes for `DeploySuperchain.t.sol`. You can see there that we do have a comment explaining why we etch

https://github.com/ethereum-optimism/optimism/blob/8a2904f51a7f6106b91408c1fae9802459c22fb3/packages/contracts-bedrock/test/DeploySuperchain.t.sol#L111-L112